requirements.txt

```diff
@@ -6,4 +6,5 @@ python-dateutil>=2.8.2
 pyahocorasick
 normality
 ovos-config
-quebra-frases
\ No newline at end of file
+quebra-frases
+PyICU
```

Comment:
Why was this dependency added? It appears to be related to build errors I started seeing today in neon-debos
```
2024/02/06 01:54:58 41-install-neon-core.sh dev | Collecting PyICU (from ovos-classifiers->skill-ddg==0.0.2a3->neon_core->neon_core[core_modules,pi,skills_default,skills_essential,skills_extended,skills_required])
2024/02/06 01:54:58 41-install-neon-core.sh dev |   Downloading PyICU-2.12.tar.gz (260 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 260.0/260.0 kB 11.6 MB/s eta 0:00:00
2024/02/06 01:55:13 41-install-neon-core.sh dev |   Installing build dependencies ... done
2024/02/06 01:55:14 41-install-neon-core.sh dev |   Getting requirements to build wheel ... error
2024/02/06 01:55:14 41-install-neon-core.sh dev |   error: subprocess-exited-with-error
2024/02/06 01:55:14 41-install-neon-core.sh dev |   
2024/02/06 01:55:14 41-install-neon-core.sh dev |   × Getting requirements to build wheel did not run successfully.
2024/02/06 01:55:14 41-install-neon-core.sh dev |   │ exit code: 1
2024/02/06 01:55:14 41-install-neon-core.sh dev |   ╰─> [58 lines of output]
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Package icu-i18n was not found in the pkg-config search path.
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Perhaps you should add the directory containing `icu-i18n.pc'
2024/02/06 01:55:14 41-install-neon-core.sh dev |       to the PKG_CONFIG_PATH environment variable
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Package 'icu-i18n', required by 'virtual:world', not found
2024/02/06 01:55:14 41-install-neon-core.sh dev |       (running 'icu-config --version')
2024/02/06 01:55:14 41-install-neon-core.sh dev |       (running 'pkg-config --modversion icu-i18n')
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Traceback (most recent call last):
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 89, in <module>
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/os.py", line 680, in __getitem__
2024/02/06 01:55:14 41-install-neon-core.sh dev |           raise KeyError(key) from None
2024/02/06 01:55:14 41-install-neon-core.sh dev |       KeyError: 'ICU_VERSION'
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       During handling of the above exception, another exception occurred:
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Traceback (most recent call last):
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 92, in <module>
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 19, in check_output
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 421, in check_output
2024/02/06 01:55:14 41-install-neon-core.sh dev |           return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 503, in run
2024/02/06 01:55:14 41-install-neon-core.sh dev |           with Popen(*popenargs, **kwargs) as process:
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 971, in __init__
2024/02/06 01:55:14 41-install-neon-core.sh dev |           self._execute_child(args, executable, preexec_fn, close_fds,
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 1863, in _execute_child
2024/02/06 01:55:14 41-install-neon-core.sh dev |           raise child_exception_type(errno_num, err_msg, err_filename)
2024/02/06 01:55:14 41-install-neon-core.sh dev |       FileNotFoundError: [Errno 2] No such file or directory: 'icu-config'
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       During handling of the above exception, another exception occurred:
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Traceback (most recent call last):
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 96, in <module>
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 19, in check_output
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 421, in check_output
2024/02/06 01:55:14 41-install-neon-core.sh dev |           return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/usr/local/lib/python3.10/subprocess.py", line 526, in run
2024/02/06 01:55:14 41-install-neon-core.sh dev |           raise CalledProcessError(retcode, process.args,
2024/02/06 01:55:14 41-install-neon-core.sh dev |       subprocess.CalledProcessError: Command '('pkg-config', '--modversion', 'icu-i18n')' returned non-zero exit status 1.
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       During handling of the above exception, another exception occurred:
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Traceback (most recent call last):
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/home/neon/venv/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
2024/02/06 01:55:14 41-install-neon-core.sh dev |           main()
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/home/neon/venv/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
2024/02/06 01:55:14 41-install-neon-core.sh dev |           json_out['return_val'] = hook(**hook_input['kwargs'])
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/home/neon/venv/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
2024/02/06 01:55:14 41-install-neon-core.sh dev |           return hook(config_settings)
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/tmp/pip-build-env-0_jelwg5/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 325, in get_requires_for_build_wheel
2024/02/06 01:55:14 41-install-neon-core.sh dev |           return self._get_build_requires(config_settings, requirements=['wheel'])
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/tmp/pip-build-env-0_jelwg5/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 295, in _get_build_requires
2024/02/06 01:55:14 41-install-neon-core.sh dev |           self.run_setup()
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "/tmp/pip-build-env-0_jelwg5/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 311, in run_setup
2024/02/06 01:55:14 41-install-neon-core.sh dev |           exec(code, locals())
2024/02/06 01:55:14 41-install-neon-core.sh dev |         File "<string>", line 99, in <module>
2024/02/06 01:55:14 41-install-neon-core.sh dev |       RuntimeError:
2024/02/06 01:55:14 41-install-neon-core.sh dev |       Please install pkg-config on your system or set the ICU_VERSION environment
2024/02/06 01:55:14 41-install-neon-core.sh dev |       variable to the version of ICU you have installed.
2024/02/06 01:55:14 41-install-neon-core.sh dev |       
2024/02/06 01:55:14 41-install-neon-core.sh dev |       [end of output]
2024/02/06 01:55:14 41-install-neon-core.sh dev |   
2024/02/06 01:55:14 41-install-neon-core.sh dev |   note: This error originates from a subprocess, and is likely not a problem with pip.
2024/02/06 01:55:14 41-install-neon-core.sh dev | error: subprocess-exited-with-error
2024/02/06 01:55:14 41-install-neon-core.sh dev | 
2024/02/06 01:55:14 41-install-neon-core.sh dev | × Getting requirements to build wheel did not run successfully.
2024/02/06 01:55:14 41-install-neon-core.sh dev | │ exit code: 1
2024/02/06 01:55:14 41-install-neon-core.sh dev | ╰─> See above for output.
2024/02/06 01:55:14 41-install-neon-core.sh dev | 
2024/02/06 01:55:14 41-install-neon-core.sh dev | note: This error originates from a subprocess, and is likely not a problem with pip.
2024/02/06 01:55:16 41-install-neon-core.sh dev | 
2024/02/06 01:55:16 Action `recipe` failed at stage Run, error: exit status 11
```